src/freenas-installer/etc/install_sata_dom.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/sh
```

Comment:
I need to confirm that but it has been so long we dropped graid support I am thinking its time we remove this code.